m4/check_darwin_paths.m4

```diff
@@ -5,8 +5,16 @@ AC_DEFUN([CHECK_DARWIN_PATHS],
 
 case $target_os in
 *darwin*)
-	CPPFLAGS="${CPPFLAGS} -I/usr/local/opt/openssl/include"
-	LDFLAGS="${LDFLAGS} -L/usr/local/opt/openssl/lib"
-	;;
+  case "$(uname -m)" in
+    arm64)
+      CPPFLAGS="${CPPFLAGS} -I/opt/homebrew/opt/openssl/include"
+      LDFLAGS="${LDFLAGS} -L/opt/homebrew/opt/openssl/lib"
+      ;;
+    **)
```

Comment:
I think we only want one `*` here, unless I'm missing something special about m4 or sh on macOS.

(Also, it looks like we indent m4 files with tabs, rather than spaces.)

I can easily fix both of these with rebasing.  Thanks again for the patch!

Reply:
I’ll fix both of those tomorrow. I want that tarsnap contributor status! :)

Reply:
ping?

I'm happy to make a new PR for these, rebased onto master.  You'll still the author of the git commits, so your tarsnap contributor status should be safe.  :)